webui/src/lib/components/auth/layout.tsx

```diff
@@ -9,11 +9,15 @@ import Card from "react-bootstrap/Card";
 import {Link} from "../nav";
 import {useLoginConfigContext} from "../../hooks/conf";
 import {useLayoutOutletContext} from "../layout";
+import Alert from "react-bootstrap/Alert";
+import {InfoIcon} from "@primer/octicons-react";
 
 type AuthOutletContext = [(tab: string) => void];
 
 
 export const AuthLayout = () => {
+    const rbacDismissedKey = "lakefs:ui:acl:showRBACAlert";
```

Comment:
```suggestion
    const rbacDismissedKey = "lakefs:ui:acl:dismissRBACAlert";
```
@ozkatz ok with you?